### ISSUE TITLE
fno_block, error when `use_mlp=True` but when `mlp=None`

### ISSUE THREAD

**Issue by christopher-beckham** [neuraloperator/neuraloperator#113]
Hi,

If the following block is created:

```
from neuralop.models.fno_block import FNOBlocks
tmp = FNOBlocks(5, 5, 10, output_scaling_factor=None, use_mlp=True)
```

The following error is raised because `mlp` needs to be populated with the args `dropout` and `expansion`:

```
Traceback (most recent call last):
  File "/network/scratch/b/beckhamc/github/NO_Diffusion/train.py", line 34, in <module>
    from models import UNO
  File "/network/scratch/b/beckhamc/github/NO_Diffusion/models.py", line 17, in <module>
    tmp = FNOBlocks(5, 5, 10, output_scaling_factor=None, use_mlp=True)
  File "/home/mila/b/beckhamc/github/neuraloperator/neuralop/models/fno_block.py", line 75, in __init__
    [MLP(in_channels=self.out_channels,
  File "/home/mila/b/beckhamc/github/neuraloperator/neuralop/models/fno_block.py", line 76, in <listcomp>
    hidden_channels=int(round(self.out_channels*mlp['expansion'])),
TypeError: 'NoneType' object is not subscriptable
```

Also see: https://github.com/neuraloperator/neuraloperator/blob/main/neuralop/models/fno_block.py#L73-L78

In the case that `mlp=None`, should the constructor internally generate a default dictionary for this? Is `{'expansion': 1, 'dropout': 0}` a reasonable default?

Thanks,
Chris

**Comment by christopher-beckham** [comment]
Perhaps if `use_mlp=True` but `mlp=None` the `__init__` constructor can just internally set a dictionary? e.g.

```
if mlp is None:
  mlp = dict(expansion=0.5, dropout=0)
```

Otherwise I think just raising an `Exception` with a nice error message could be a viable alternative.

**Comment by JeanKossaifi** [comment]
Fixed in #143
